tools/harness-simulation/posix/harness_dev_discovery.py

```diff
@@ -145,10 +161,27 @@ def main():
     # Get the local IP address on the specified interface
     addr = get_ipaddr(args.ifname)
 
+    # Initiate the sniffer
```

Comment:
initialize?

Reply:
The difference between **initialize** and **initiate** is that **initialize** is to assign initial values to something while **initiate** is to begin, to start.

I think using **initiate** is more proper here. The following statements not only initialize the variable `sniffer_procs`, but also start several processes of sniffer simulation servers.